src/Conn.h

```diff
@@ -37,16 +37,37 @@ typedef enum {
 	NUM_EVENTS_TO_FLAG,
 } ConnEventToFlag;
 
+typedef enum {
+	BASIC,
+	BASIC_WITH_VLAN
+} ConnIDType;
+
 typedef void (Connection::*timer_func)(double t);
 
 struct ConnID {
+	void* data;
+	unsigned int type;
+};
```

Comment:
Don't think we need to store the type here -- since there's only a single/static ConnID mode per run, can just always inspect the global ConnID mode we're using rather than store it per ConnID instance.

The `void*` also seems to make for some awkward casting moves, but not sure what to suggest as optimal.  Maybe until there's more feedback regarding what ConnID modes to support, it can just be an over-specified type that includes any possible field, which we then pick and choose from during the session-izing hashing/comparison operations.